lib/rollbax.ex

```diff
@@ -25,6 +25,11 @@ defmodule Rollbax do
       are reported; if `false`, `Rollbax.report/5` is basically a no-op; if
       `:log`, exceptions reported with `Rollbax.report/5` are instead logged to
       the shell.
+    * `:custom` (map) a map of any arbitrary metadata you want to attach to
```

Comment:
Sorry for the extreme nitpicking™, missing a "-" here between the option name and its type :)